src/ngraph/distributed.hpp

```diff
@@ -24,6 +24,16 @@
 
 namespace ngraph
 {
+
+    enum class Reduce_t
```

Comment:
We usually don't use the `_t` suffix in nGraph.

Reply:
I create another `reducetion::Type`, like following `element::Type`, but both use `Type_t`

Reply:
I don't think `element::Type_t` should be called `element::Type_t`, but I have to admit it establishes a precedent. :)